src/common/Pressable/styles.ts

```diff
@@ -0,0 +1,18 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  commonContainer: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    minHeight: 44,
+    minWidth: 44,
```

Comment:
Is it needed to force a `minHeight` and `minWidth` for this generic `Pressable` component? I'm thinking of it as a wrapper for `react-native`'s `Pressable` component with an opacity feedback, so maybe having a pre-defined style is not desired.

Same question applies to `alignItems` and `justifyContent`.

Reply:
These props are to follow the suggested accessibility statement which says all pressable buttons should have at least 44px, (you can read more [here](https://www.accessguide.io/guide/large-target-size#:~:text=Large%20buttons%20(44%20px)%20are,interact%20with%20websites%20and%20apps.)). You can always change it if you don't need it but I think it'd be a good guide to have it as a default value in our base.